### ISSUE TITLE
Multimedia keyboard shortcuts stopped working with last updates

### ISSUE THREAD

**Issue by derkomai** [elementary/switchboard-plug-keyboard#213]
Since last week's updates (May ~2th-3th), I am not able to use multimedia shortcuts in any of my pcs anymore. I can reproduce this with several music players.

**Comment by derkomai** [comment]
Super + L is also not working for "Lock session".

**Comment by peteruithoven** [comment]
Thanks for the report! I'm afraid this is a duplicate of https://github.com/elementary/os/issues/198

**Comment by peteruithoven** [comment]
@derkomai 
Are you running elementary OS Juno?
What happens when you restart? You still have the issue? If I understand you correctly your issue is actually different from elementary/os#198 because you experience this permanently?
What happens when you restart `gsd-media-keys` by running the following?
```
killall gsd-media-keys
```

**Comment by derkomai** [comment]
Yes, I'm running Juno. I think you're right. After rebooting everything seems working again. I thought I had rebooted but it could be that I hadn't, I'm not sure. Sorry about the trouble.

**Comment by peteruithoven** [comment]
Ah thanks for checking. In that case it's a duplicate of: elementary/gala#193
